charts/tuf/values.yaml

```diff
@@ -15,15 +15,22 @@ deployment:
   port: 8080
 
 secrets:
+  create: false
   rekor:
     name: rekor-public-key
+    key: public
     path: rekor-pubkey
+    value: b3B0aW9uYWw=
```

Comment:
Q: I assume you don't need to set the value here.

Reply:
Yup, changed after discussion with Andrew